Oracle/pcornet_mapping.csv

```diff
@@ -89,11 +89,11 @@ pcori_path,local_path
 \PCORI\DIAGNOSIS\DX_TYPE\09\,\i2b2\Diagnoses\
 \PCORI\PROCEDURE\PX_TYPE\C4\,\i2b2\Procedures\PRC\Metathesaurus CPT Hierarchical Terms\
 \PCORI\PROCEDURE\PX_TYPE\09\,\i2b2\Procedures\PRC\ICD9 (Inpatient)\
-\PCORI\VITAL\HT\,\i2b2\Flowsheet\KU\GEN CARD\IP/OP HEIGHT/WEIGHT T:900445\KU GEN CARD G IP/OP PROCEDURE HEIGHT/WEI G:900612 I#1\HEIGHT M:11 I#1\
```

Comment:
It was made very clear to me from the meetings yesterday that flowsheets were absolutely the **wrong** choice for height/weight (or, flowsheets alone anyway).

I lean towards sticking with just the vitals for simplicity, but we can certainly add both paths with a little more validation/on-going maintenance.  Doing so does apparently cover more encounters.

Here are the pros/cons I see (using weight as an example):

**Weight from flowsheets and vitals**
Pros:
 - We cover more encounters - apparently, about 20% more

Cons:
 - Extra validation/on-going maintenance
  - There are 47 different flowsheet paths that have exactly the same fact/patient count - nearly 750 more paths with fewer counts.  We'd have to validate we don't end up with an explosion of facts due to duplicates when match against so many paths during the CDM transform.  Also, I'd think we'd want to check all the paths with every CDM ETL and change our .csv as necessary to match the hospital changes (maybe a good idea anyway).

**Weight from vitals only**
Pros:
 - Simple, hopefully static path that catches all the patients (well, only a difference of 31 from flowsheets). 

Cons:
 - We miss out on encounters (~20%).  That's based on our encounter_num - it's not clear to me whether or not they really are different encounters (that gets in to our encounter design we're evaluating).
 

Reference queries:
```
-- Weight related paths
select c_name, c_totalnum, c_fullname 
from blueheronmetadata.heron_terms 
where lower(c_name) like '%weight%' and 
c_totalnum is not null 
--and c_fullname like '\i2b2\Flowsheet\%'
order by c_totalnum desc;

-- Distinct encounters
with codes as (
  select concept_cd 
  from blueherondata.concept_dimension 
  -- Picking one of the 47 paths with the same count
  where concept_path like '\i2b2\Flowsheet\KU\ONC\TMP TRT FLOWSHEET T:714\KU ONC GRP TRT FLOW VITALS/SCREENING G:15002 I#1\WEIGHT/SCALE M:14 I#13\%'
  -- Just vitals
  --where concept_path like '\i2b2\Visit Details\Vitals\WEIGHT\%'
  )
select count(distinct encounter_num)
from blueherondata.observation_fact
join codes on codes.concept_cd = observation_fact.concept_cd;
```